### PR TITLE
Bug 1490400 Have telemetry.sh exit on failed commands

### PR DESCRIPTION
**Pull request by jklukas**
The existing comment about `set -e` appears to no longer apply.
I've been deploying clusters while iterating on these changes,
hitting many "Bootstrap failure" states. I have reliably been
able to find stdout.log.gz containing the relevant failure message
for these failed clusters.

There were a few commands failing for every cluster, so I had to
make some small cleanup changes in order for bootstrap to complete
with the new `set -eo pipefail` setting.

### CODE REVIEW

File: ansible/files/bootstrap/telemetry.sh [hunk 1]
```diff
@@ -243,11 +249,12 @@ fi
 
 # Setup R environment
 cd /mnt
-wget -nc https://mran.microsoft.com/install/RRO-3.2.1-el6.x86_64.tar.gz
```
Comment (jklukas) [inline ansible/files/bootstrap/telemetry.sh]: This URL was returning 404

**Comment by jklukas** [comment]
Also, I've spun up a cluster via atmo.stage and confirmed that telemetry hello world and longitudinal notebooks run successfully with these changes.

File: ansible/files/bootstrap/telemetry.sh (hunk 1 above)
Comment (sunahsuh) [reply ansible/files/bootstrap/telemetry.sh]: I guess that begs the question – if it broke and nobody complained about it, should we keep it?

Comment (jklukas) [reply ansible/files/bootstrap/telemetry.sh]: That is a valid question and should be a follow-up here, but likely it makes sense to have that conversation in a batch format. I can write up a follow-up bug to identify seemingly unused bits in here and have a mailing list or Slack discussion about what we can remove.

Comment (jklukas) [reply ansible/files/bootstrap/telemetry.sh]: Filed https://bugzilla.mozilla.org/show_bug.cgi?id=1513992

**Pull request merged**